src/stories/ErrorBox.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import { Story } from '@storybook/react';
+import React from 'react';
+import ErrorBox, { Props as ErrorBoxProps } from '../components/ErrorBox/ErrorBox';
+
+export default {
+  title: 'ErrorBox',
+  component: ErrorBox,
+};
+
+const Template: Story<ErrorBoxProps> = (args) => {
+  return (
+    <ErrorBox
+      {...args}
+      text={'This is an error message'}
```

Comment:
Yo don't need to pass this props here because you are already passing them by doing {...args}. Also, if you change values on the fly on storybook they are not going to change if you have them harcoded here

Reply:
Thanks, makes sense... will fix.